frontend/form-builder/src/components/FormBoard/FormBoard.tsx

```diff
@@ -1,22 +1,60 @@
 import { Heading, Paragraph } from "@digdir/design-system-react";
 import React from "react";
-import { ComponentIcon } from "@navikt/aksel-icons";
+import { ComponentIcon, XMarkIcon } from "@navikt/aksel-icons";
 import classes from "./FormBoard.module.css";
 import { useTranslation } from "react-i18next";
+import { useDrop } from "react-dnd";
+import { DraggableItemsType } from "../../types/dndTypes";
+import { ItemProps } from "../Toolbar/DraggableItem/DraggableItem";
```

Comment:
Possible to shorten import path by including ItemProps in index.ts.
```suggestion
import { ItemProps } from "../Toolbar/DraggableItem";
```